src/genie/libs/parser/iosxe/tests/ShowLispEidTableVrfUserIpv4MapCache/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,541 @@
+expected_output = {
+    "vrf": {
+        "User": {
+            "iid": 4100,
+            "number_of_entries": 2186,
+            "eid": {
+                "0.0.0.0/0": {
+                    "uptime": "1w6d",
+                    "expire": "never",
+                    "via": [
+                        "static-send-map-request"
+                    ],
+                    "rloc": {
+                        "status": "Negative cache entry",
+                        "action": "send-map-request"
+                    }
+                },
+                "0.0.0.0/5": {
+                    "uptime": "4w6d",
+                    "expire": "00:14:38",
+                    "via": [
+                        "map-reply",
+                        "forward-native"
+                    ],
+                    "rloc": {
+                        "status": "encapsulating_to_proxy_etr"
+                    }
+                },
+                "8.0.0.0/7": {
+                    "uptime": "07:18:55",
+                    "expire": "00:07:19",
+                    "via": [
+                        "map-reply",
+                        "forward-native"
+                    ],
+                    "rloc": {
+                        "status": "encapsulating_to_proxy_etr"
+                    }
+                },
+                "10.0.0.0/9": {
+                    "uptime": "1w4d",
+                    "expire": "00:00:13",
+                    "via": [
+                        "map-reply",
+                        "forward-native"
+                    ],
+                    "rloc": {
+                        "status": "encapsulating_to_proxy_etr"
+                    }
+                },
+                "10.128.0.0/11": {
+                    "uptime": "4w6d",
+                    "expire": "00:04:45",
+                    "via": [
+                        "map-reply",
+                        "forward-native"
+                    ],
+                    "rloc": {
+                        "status": "encapsulating_to_proxy_etr"
```

Comment:
sorry, it seemed I missed to comment on this. please take the value as is.
```suggestion
                        "status": "Encapsulating to proxy ETR"
```

Reply:
Updated.